components/zeekendpointsecurity/src/endpointsecurityconsumer.cpp

```diff
@@ -131,6 +132,10 @@ void EndpointSecurityConsumer::endpointSecurityCallback(
 
   } else if (message.event_type == ES_EVENT_TYPE_NOTIFY_FORK) {
     status = processForkNotification(event, message_ptr);
+  } else if (message.event_type == ES_EVENT_TYPE_NOTIFY_OPEN) {
+    status = processOpenNotification(event, message_ptr);
+  } else if (message.event_type == ES_EVENT_TYPE_NOTIFY_CREATE) {
+    status = processCreateNotification(event, message_ptr);
   }
```

Comment:
While you are at it, can you make this block use a `switch` since we always plan to handle all cases?